### ISSUE TITLE
Autocompletion for variables from python variable files

### ISSUE THREAD

**Issue by JaPyR** [robocorp/robotframework-lsp#333]
Not sure whether it bug or feature request, but at least in my case autocompletion isn't working for variables from `.py` files.

Minimal example:

```python
# vars.py
SOME_VAR = "ABC"
```

```robotframework
*** Settings ***
Variables    vars.py

*** Test Cases ***
Demo
    Log    ${SO}    # I would expect autocompletion to work here
```

**Comment by JaPyR** [comment]
@fabioz In theory this issue could be closed as in general autocompletion for variables from python files works, but `__all__` isn't respected (as robotframework does https://robotframework.org/robotframework/latest/RobotFrameworkUserGuide.html#variable-files). 
Here is an example:
```python
# vars.py
__all__ = ["FOO"]

FOO = "Bar"
HELLO = "WORLD"
```

```robotframework
*** Settings ***
Variables    vars.py

*** Test Cases ***
DemoVars
    Log    ${FOO}
    Log    ${HE}
```

and a screenshot
![image](https://user-images.githubusercontent.com/7773301/157872792-f9ece70d-0bf1-441b-bb57-0d2fdf5a79ff.png)

**Comment by fabioz** [comment]
Yes, the auto-completion was actually implemented for Python files in another issue. I'll repurpose this ticket to respect the `__all__`.
